proposal/numbers.md

```diff
@@ -0,0 +1,104 @@
+# Floating Point Numbers: Draft 1.0
+
+*[(Issue)](https://github.com/sass/sass/issues/2892)*
+
+This proposal formally defines all numbers in Sass to be double-precision or 64-bit floating point numbers.
+
+## Table of Contents
```

Comment:
This should be structured following the format listed [here](https://github.com/sass/sass/blob/master/CONTRIBUTING.md#proposal). Most of the sections here should be subsections under "Semnantics" and you should add a "Summary" section.

Reply:
Ah my bad! I did not see that document. Thank you -- I have reformatted the document to include "Semantics" and "Summary" sections